### PR TITLE
Fix: Issue #770 scrollUp button

### LINKED ISSUE #770: Adding button to scroll-up(top) in "Show All Challenges" view
Ref : https://material.angularjs.org/latest/ .
A button at the right-bottom corner which will smooth scroll up.

### PR DESCRIPTION
**Pull request by Obtin**
Fix #770: Adding button to scroll-up(top) in "Show All Challenges" view

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@Obtin Can you please paste the screenshot here?

**Comment by Obtin** [comment]
@RishabhJain2018 
scroll

**Comment by RishabhJain2018** [comment]
@Obtin the button works perfectly! but can you please make this button circular and right aligned.
@aka-jain @spyshiv Please take a look once!

**Comment by Obtin** [comment]
@RishabhJain2018,
scroll2
 is it okay now?

**Comment by RishabhJain2018** [comment]
@Obtin The design looks good now :-), But it isn't working on my local. Can you please check once?

**Comment by RishabhJain2018** [comment]
@Obtin Can you please paste the screenshot of the updated button?

**Comment by Obtin** [comment]
@RishabhJain2018 
shadow:
srollupr1
on cursor:
scrollup3

File: frontend/src/js/controllers/challengeListCtrl.js [hunk 1]
```diff
@@ -147,7 +147,16 @@
 
         utilities.sendRequest(parameters);
 
-
+        vm.scrollUp = function() { 
+            utilities.hideButton();
+            angular.element($window).bind('scroll', function(){
+                if(this.pageYOffset >= 100 ){
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/challengeListCtrl.js]: Please change `this.pageYOffset` to `this.pageYoffset`.

**Comment by Obtin** [comment]
@RishabhJain2018 done!!

**Pull request merged**